tests/Integration/Behaviour/Features/Context/Domain/AliasFeatureContext.php

```diff
@@ -0,0 +1,127 @@
+<?php
+/**
+ * Copyright since 2007 PrestaShop SA and Contributors
+ * PrestaShop is an International Registered Trademark & Property of PrestaShop SA
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.md.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://devdocs.prestashop.com/ for more information.
+ *
+ * @author    PrestaShop SA and Contributors <contact@prestashop.com>
+ * @copyright Since 2007 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ */
+
+declare(strict_types=1);
+
+namespace Tests\Integration\Behaviour\Features\Context\Domain;
+
+use Behat\Gherkin\Node\TableNode;
+use Doctrine\DBAL\Driver\Exception;
+use PHPUnit\Framework\Assert;
+use PrestaShop\PrestaShop\Core\Domain\Alias\Command\AddAliasCommand;
+use PrestaShop\PrestaShop\Core\Grid\Query\AliasQueryBuilder;
+use PrestaShop\PrestaShop\Core\Search\Filters\AliasFilters;
+use Tests\Resources\DatabaseDump;
+
+class AliasFeatureContext extends AbstractDomainFeatureContext
+{
+    /**
+     * @When I add alias with following information:
+     *
+     * @param TableNode $table
+     */
+    public function addAlias(TableNode $table): void
+    {
+        $data = $table->getRowsHash();
+
+        $aliases = array_map('trim', explode(',', $data['alias']));
```

Comment:
I think this should work
```suggestion
        $aliases = PrimitiveUtils::castStringArrayIntoArray($data['alias'])
```

Reply:
and i think the variable is redunant as well, see suggestion bellow